src/ucp/rma/basic_rma.c

```diff
@@ -281,6 +297,8 @@ ucs_status_t ucp_get_nbi(ucp_ep_h ep, void *buffer, size_t length,
     uct_rkey_t uct_rkey;
     ucs_status_t status;
 
+    UCP_THREAD_CS_ENTER(ep->worker->mt_config);
```

Comment:
actually now i see that all is ok (though that the code belonged to progress_get_nbi, while actually it belongs to get_nbi). so pls disregard this comment.